platform/darwin/src/MGLStyle.h

```diff
@@ -258,6 +258,21 @@ static const NSInteger MGLStyleDefaultVersion = 9;
  */
 - (void)removeStyleClass:(NSString *)styleClass;
 
+/**
+ Sets an image to the sprite atlas with a unique identifier.
```

Comment:
“Sprite atlas” is jargon that most developers would find unintuitive. It’s probably sufficient to describe the image being added to the style (since it lasts until the style URL changes), then mention that the developer can use the image by setting the `MGLSymbolStyleLayer.iconImage` property to the name passed in here.